src/Cache.php

```diff
@@ -4,29 +4,39 @@
 
 namespace Interns2022B;
 
+use Illuminate\Support\Collection;
+use Interns2022B\Models\Brewery;
+
 class Cache
 {
     public const CLEAR = "clear";
     public const BUILD = "build";
     public const TABLE_DIRECTORY = __DIR__ . "/../tests/stubs/table.json";
 
-    public array $toFile = [];
+    private array $toFile = [];
 
-    public function putData(string $provider, array $breweriesData): void
+    public function putData(string $provider, Collection $breweriesFactory): void
```

Comment:
I would like to rename breweriesFactory to breweries
Let's add doc above method name: /** @var Collection<Brewery> $breweries */

Reply:
Use `@param` instead of `@var`. My bad.